.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+    - "2.7"
+    - "3.3"
+install:
+    pip install -r requirements.txt -e .
+script: nosetests
```

Comment:
Do we have to chmod the file before we can call it in Travis? If that's the fix, let me tackle that in a different PR.